src/docs/product/sentry-basics/tracing/event-detail.mdx

```diff
@@ -62,16 +62,19 @@ The span view may be limited to one project at a time if you're on the [Team pla
 
 </Alert>
 
-#### Traversing Transactions
```

Comment:
@doralchan I don't think we should delete the traversal portion of the content. I feel that embedded transaction and traversal are two distinct (but similar) product features that have some value.

Reply:
That was me. I have a potential solution that we can go over in our meeting this afternoon.